### DISCUSSION TITLE
Question: relating to Bopeebo

### DISCUSSION THREAD

**Discussion by Iamthebestintheworld222** [FunkinCrew/Funkin#4018] [Q&A] [answered]
[weed]: <> (This isn't a place for AMA type questions, if you want to ask any of the devs something, reach out to them on twitter prob )
[weed]: <> (any biz bullshit can go to cameron.taylor.ninja@gmail.com)

**Comment by Hundrec** [comment, marked as answer]
It's been a long time since this was posted, but I'll say that they probably could have kept BF's voices the way they were before 0.3.0.
They likely made this change due to the positive reception for the older (now current) version of Bopeebo.

The only other possibility I can think of is that Kawaisprite lost the project file for the vibrato version of Bopeebo.
